### PR TITLE
Add contributing details

### PR DESCRIPTION
**Pull request by calebbrown**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,11 +9,146 @@ We'd love to accept your patches and contributions to this project.
...
+    - From the repo run (requires [Go](https://go.dev/doc/install)):
+
+    ```shell
+    go run ./cmd/ingest \
+       -config config/config.yaml \
+       -dir /path/to/osv \
+       -source [source id]
+    ```
+
+    - Fix any issues. You will need to revert the repo before running
```
Comment (oliverchang) [inline CONTRIBUTING.md:87]: Nit: perhaps we can avoid overloading the use of "issues" here and use an alternative like "problems reported" or something.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,11 +9,146 @@ We'd love to accept your patches and contributions to this project.
...
+      ingest again.
+1. Commit and push the changes to a form and submit a Pull Request.
+    - Reference the issue from step #1 in the PR.
```
Comment (oliverchang) [inline CONTRIBUTING.md:90]: What if someone wants to do periodic bulk imports? Would they reference the same issue each time?  And that issue would stay open?

**Review by oliverchang — COMMENTED** [review]
nice!

File: CONTRIBUTING.md (hunk 1 above)
Comment (calebbrown) [reply CONTRIBUTING.md]: Fixed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (calebbrown) [reply CONTRIBUTING.md]: I've tried to clarify this. Let me know what you think.

**Review by oliverchang — APPROVED** [review]

**Comment by calebbrown** [comment]
Ping @jossef

**Pull request merged**